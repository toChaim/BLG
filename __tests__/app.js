const request = require('supertest');
const app = require('../app');

describe('Test App', () => {
  describe('GET / ', () => {
    test('Should return home page', async () => {
      const res = await request(app).get('/');
      expect(res.statusCode).toBe(200);
      expect(res.type).toBe('text/html');
      expect(res.body).toMatchSnapshot();
    });
  });
});
