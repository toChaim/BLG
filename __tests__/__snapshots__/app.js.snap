// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /  Should return home page 1`] = `Object {}`;
